data/mods/Magiclysm/items/spell_scrolls.json

```
@@ -775,6 +793,105 @@
+    "id": "spell_scroll_air_bubble",
+    "//": "Stormshaper spell",
+    "name": { "str": "Scroll of Air Bubble", "str_pl": "Scrolls of Air Bubble" },
+    "description": "Cover your head with an air bubble, allowing you to breathe underwater or prevent breathing of toxic gases.",
+    "use_action": { "type": "learn_spell", "spells": [ "air_bubble" ] }
+  },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_storm_superconductor",
+    "//": "Stormshaper spell",
+    "name": { "str": "Scroll of Superconductor", "str_pl": "Scrolls of Superconductor" },
+    "description": "Create a devastating electric charge on a small distance, roasting everything to their death.",
+    "use_action": { "type": "learn_spell", "spells": [ "storm_superconductor" ] }
+  },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_shocking_dash",
+    "//": "Stormshaper spell",
+    "name": { "str": "Scroll of Shocking Dash", "str_pl": "Scrolls of Shocking Dash" },
+    "description": "Fill your legs with energy, and dash into picked place.",
+    "use_action": { "type": "learn_spell", "spells": [ "shocking_dash" ] }
+  },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_toxtricity",
+    "//": "Stormshaper spell",
+    "name": { "str": "Scroll of Toxtricity", "str_pl": "Scrolls of Toxtricity" },
+    "description": "Inject an electric energy into the creatures, which makes them weak, and deal huge damage over time.",
+    "use_action": { "type": "learn_spell", "spells": [ "toxtricity" ] }
+  },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_electric_eye",
+    "//": "Stormshaper spell",
+    "name": { "str": "Scroll of Electric Eye", "str_pl": "Scrolls of Electric Eye" },
+    "description": "Enhance your sight with your powers.  It allows you to see creatures of the same energy around you, and block or dodge an additional attack your enhanced perception speed could anticipate.",
+    "use_action": { "type": "learn_spell", "spells": [ "electric_eye" ] }
+  },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_electric_waves",
+    "//": "Stormshaper spell",
+    "name": { "str": "Scroll of Electric Waves", "str_pl": "Scrolls of Electric Waves" },
+    "description": "Your hands start to emit a huge amount of energy, that you release with each attack.",
+    "use_action": { "type": "learn_spell", "spells": [ "electric_waves" ] }
+  },
+  {
+    "type": "BOOK",
+    "copy-from": "spell_scroll",
+    "id": "spell_scroll_electric_arts",
+    "//": "Stormshaper spell",
+    "name": { "str": "Scroll of Electric Arts", "str_pl": "Scrolls of Electric Arts" },
+    "description": "This spell were designed for scholars, that just started to learn stormshaping.  It allows them to use their abilities, when their magic control is weak, and they can injure themselves with electricity.",
```

```suggestion
    "description": "Insulate the skin against electrical damage. Particularly useful for the apprentice Stormshaper, whose  magical control is weak and whose efforts could easily end in self-electrocution.",
```
Imma sick @FoolsGold45 on your descriptions if you use "this [thing you're obviously looking at]" 😛